source-kinesis/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Build Stage
+################################################################################
+FROM golang:1.17-buster as builder
+
+WORKDIR /builder
+
+# Compile-time dependencies
+# RUN apt-get update \
+#  && apt-get install -y \
+#       ca-certificates \
+#  && apt-get clean
+
+# Download & compile dependencies. Doing this separately allows caching this
+# layer when no dependencies are updated.
+COPY go.* /builder/
```

Comment:
nit: /builder/ is already the WORKDIR.

`COPY go.* ./` here, and relative paths elsewhere in this Dockerfile ?